packages/worker/native/src/lib.rs

```diff
@@ -13,16 +16,20 @@ use temporal_sdk_core::{
 
 type BoxedWorker = JsBox<Arc<Worker>>;
 
+pub struct PollItem {
+    queue_name: String,
+    callback: Root<JsFunction>,
+}
+
 pub struct Worker {
     core: Box<dyn Core + Send + Sync>,
-    condition: Condvar,
-    suspended: Mutex<bool>,
+    sender: SyncSender<PollItem>,
```

Comment:
A docstring about what this is for would be nice

In fact, generally I think we could use a lot more docstrings throughout. These libs are complex and a little explanation goes a long way.